Tests/varLib/instancer_test.py

```diff
@@ -279,57 +299,81 @@ def test_scaleVarDataDeltas(self):
 
         instancer._scaleVarDataDeltas(varData, regionScalars)
 
-        assert varData.Item == [[50, 0], [-50, 0]]
+        assert varData.Item == [[50], [-50]]
+
+    def test_popVarDataDeltas(self):
+        varData = builder.buildVarData([1, 0], [[34, 68], [-100, -200]], optimize=False)
+        assert instancer._popVarDataDeltas(varData, 1) == [34, -100]
+        assert varData.VarRegionCount == 1
+
+        varData = builder.buildVarData([1, 0], [[34, 68], [-100, -200]], optimize=False)
+        assert instancer._popVarDataDeltas(varData, 0) == [68, -200]
+        assert varData.VarRegionCount == 1
+
+        varData = builder.buildVarData([1, 0], [[34, 68], [-100, -200]], optimize=False)
+        assert instancer._popVarDataDeltas(varData, 2) == [0, 0]  # missing
+        assert varData.VarRegionCount == 2
 
-    def test_getVarDataDeltasForRegions(self):
+    def test_mergeVarDataRegions(self):
         varData = builder.buildVarData(
-            [1, 0], [[33.5, 67.9], [-100, -200]], optimize=False
+            [3, 1, 2, 0, 4], [[12, 34, 56, 78, 0], [91, 23, 45, 67, -1]]
         )
 
-        assert instancer._getVarDataDeltasForRegions(varData, {1}) == [[33.5], [-100]]
-        assert instancer._getVarDataDeltasForRegions(varData, {0}) == [[67.9], [-200]]
-        assert instancer._getVarDataDeltasForRegions(varData, set()) == [[], []]
-        assert instancer._getVarDataDeltasForRegions(varData, {1}, rounded=True) == [
-            [34],
-            [-100],
-        ]
+        instancer._mergeVarDataRegions(varData, [0, 0, 0, 3, 4])
```

Comment:
Can you add another test case where the number of regions referenced in varData is less than the actual number of regions:

varData = builder.buildVarData([3, 1, 2, 0, 4], [[12, 34, 56, 78, 0], [91, 23, 45, 67, -1]])
instancer._mergeVarDataRegions(varData, [0, 0, 0, 3, 4, **5**])